### ISSUE TITLE
 cert_verify.sh in dos format

### ISSUE THREAD

**Issue by NgSekLong** [mmumshad/kubernetes-the-hard-way#203]
For some reason my `cert_verify.sh` is in DOS format, so bash cert_verify.sh by default don't work.

I need to execute the following to fix this:
```
sudo apt-get update
sudo apt-get install -y dos2unix
dox2unix cert_verify.sh
```
Before executing `bash cert_verify.sh`

Not sure if other encountered this issue as well.

If other also encountered this issue, I can submit a PR to add this comment to the doc, or this issue might be able to fix during the vagrant deployment, cheers.

**Comment by bonjovi27** [comment]
thanks a lot.
I having the same issue. now it works fine
![image](https://user-images.githubusercontent.com/37484975/123677153-694d4b80-d862-11eb-9800-2df64c31f8c0.png)

**Comment by fireflycons** [comment]
Hi @NgSekLong  @bonjovi27 

We have now merged a major change to bring this to v1.24.
Many issues, including this, have been addressed as part of the upgrade. Please try it now and feel free to raise further issues.

Thanks.
